R/utils.R

```diff
@@ -162,3 +205,36 @@ modes <- function() {
     "worst_case"
   )
 }
+
+#' Scenarios
+#'
+#' @keywords internal
+#' @export
+scenarios <- function() {
+  c(
+    "1.5C RPS",
+    "NZ 2050"
+  )
+}
```

Comment:
Nice!

Just FYI ...

By discussing how to make our code robust to changes you helped me remember about the [config](https://rstudio.github.io/config/) package, which I'm using for the same reasons in the tiltWebTool package.

* [All brittle details are stored in a configuration file](https://github.com/2DegreesInvesting/tiltWebTool/blob/main/inst/config/theme.yml)
* [A higher level helper gets the required configurations and uses wherever we need them.](https://github.com/search?q=repo%3A2DegreesInvesting%2FtiltWebTool+theme_color&type=code)

This means Anne/Tilman can themselves edit a less-scary-looking .yml file -- maybe on GitHub itself.

```yaml
# See https://rstudio.github.io/bslib/reference/bs_theme.html
default:

  bg: "#E2E5E9"
  fg: "#000000"
  primary: "#287155"
  secondary: "#B3D15D"
  success: "#B3D15D"
  info: "#5170CF"
  warning: "#F6CB4E"
  danger: "#E3693B"

  base_font: "Roboto"
  base_font_fallback: "sans-serif"
  code_font: "Source Code Pro"
  code_font_fallback: "monospace"
  heading_font: "Roboto Condensed"
  heading_font_fallback: "sans-serifx"
```

Yet another manager-friendly UI that I'm using is the good-old GoogleSheet. This is to allow Anne/Tilman to edit a datadictionary themselves, and the app just reads and prints that googlesheet. When the name of a column changes, they can themselves change it on the googlesheet. [The app just reads the googlesheet whatever it has](https://github.com/2DegreesInvesting/tiltWebTool/blob/main/R/dictionary.R). To avoid a dependency on the googlesheets4 package I publish the googlesheet to a .csv -- directly from the googlesheets File > Share menu


![image](https://github.com/2DegreesInvesting/tiltPlot/assets/5856545/39d48222-2323-4d3e-ad88-7654f2116bb9)


These tricks may not be useful now but maybe you'll remember about them in a few years from now :-)

And the general "clean architecture" idea is always the same: separate the things that change for different reasons and at different frequency -- meaning a lower level detail such a specific "background color"  must be separated from the higher level code that manipulates the "theme".

Reply:
Thank you for this insight Mauro !!!